modules/gdscript/gdscript_parser.cpp

```diff
@@ -4160,6 +4160,64 @@ static String _get_annotation_error_string(const StringName &p_annotation_name,
 	return vformat(R"("%s" annotation requires a variable of type %s, but type "%s" was given instead.)", p_annotation_name, string, p_provided_type.to_string());
 }
 
+static StringName _find_narrowest_native_or_global_class(const GDScriptParser::DataType &p_type) {
```

Comment:
Should it be "nearest"? Or is "narrowest" a term that typically has this meaning in class inheritance?

Reply:
We are moving up the inheritance hierarchy. If we find a global script class, we stop and use it. If not, then we reach the first native class and use it. "Narrowest" means we are looking for the most specific type (having a global name) as possible.